AsyncDisplayKit/ASTableView.mm

```diff
@@ -563,14 +565,23 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 
 - (ASScrollDirection)scrollDirection
 {
-  CGPoint scrollVelocity = [self.panGestureRecognizer velocityInView:self.superview];
+  CGPoint scrollVelocity;
```

Comment:
Can't wait to move this logic into the node classes, and then have a shared superclass!